ci/test.yaml

```diff
@@ -0,0 +1,9675 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
What do we need this file for?

Reply:
I removed the test file. We should remove the Jenkinsfile also. I am not sure about the cron job. @vacas5 do we need the cron job?

Reply:
As @cmastr pointed out in slack there is a part in the cronjob which cleans up the unused s3 buckets from the previews which we should remove.

I think we definitely still need that part.

But I'm still not sure about the dev-image right now.

Reply:
Looks like we do not need the dev image anymore.

Reply:
okay that makes sense. I'll try to move the cleanup to github actions as well. We dont really want anything running in jenkins